dev/com.ibm.ws.cdi.extension.bce_fat/fat/src/com/ibm/ws/cdi/extension/bce_fat/BuildCompatibleExtensionsTest.java

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.extension.bce_fat;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+import com.ibm.ws.cdi.extension.bce_fat.basic.app.BasicBCEExtension;
```

Comment:
isn't the "app" subpackage redundant? the app is called "basic"

Reply:
Good catch. Previously the `basic` package had the test and the `basic.app` package had the test app contents.

When I added the EAR test, it made sense to use one test class for both tests to avoid restarting the server.